docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -98,6 +98,16 @@ error-prone migrations, without downtime, without requiring any
 additional network traffic, and without any extensive communication with
 downstream users.
 
+With SCU, different versions of the _same_ package are now grouped under a common package-name.
+This allows template/interface IDs to now be unambiguously referenced using either:
+
+-  the package ID
+
+-  or by using the package-name (which is resolved by a Canton participant to a unambiguous package ID).
+
+When defining a Daml project, the project-name field of the project's `daml.yaml` is now used to
+specify the SCU project-name.
```

Comment:
```suggestion
When defining a Daml package, the ``name`` field of the package's ``daml.yaml`` is now used to
specify the SCU package-name.
```

Reply:
Be careful to use `package` over `project`, as we like to use project now to describe a set of packages.
Also docs.daml.com requires double backticks for code sections